guides/release/models/finding-records.md

```diff
@@ -38,7 +40,7 @@ Use [`store.peekAll()`](https://api.emberjs.com/ember-data/release/classes/Store
 let blogPosts = this.store.peekAll('blog-post'); // => no network request
 ```
 
-`store.findAll()` returns a `PromiseArray` that fulfills to a `RecordArray` and `store.peekAll` directly returns a `RecordArray`.
+`findRecord()` returns a `PromiseArray` that fulfills to a `RecordArray` and `store.peekAll` directly returns a `RecordArray`.
```

Comment:
This is not really correct here. As `findRecord` is request builder and returns options for the request to make. And then later `store.request` returns Future or StructuredDocument. Please checkout documentation for `@ember-data/request`